### PR TITLE
Support Webpack by using default import of big.js

### PR DESCRIPTION
**Pull request by chrisgarber**
### Status
**READY/IN-PROGRESS/HOLD**
Ready

### Description
**FEATURE/BUG FIX/REFACTOR**

In order to import the currency module through webpack, the default export from big.js must be used, else a _big.Big will not be a constructor.

### Related PRs

N/A

### Note to Reviewers
Please ensure that my documentation is in the correct style!

### Quality Checklist:
These are mandatory and must be completed before merging the PR

- [ ] New, changed or refactored functionality has tests.
^ I was unsure how to test module loading as it is somewhat outside the scope of this repo
- [x] New, changed or refactored functionality has documentation.
- [x] New, changed or refactored functionality has been QA'd.
- [x] New, changed or refactored functionality has been added to the CHANGELOG.

### CODE REVIEW

**Comment by iansolano** [comment]
👋@chrisgarber, thanks for submitting a PR!

Sorry you experienced an issue using `clark-utils` in your project. This is actually an issue with the way dependencies were being declared. Fixed in this PR: https://github.com/hiclark/clark-utils/pull/11.

We would like to merge your PR though, `big.js` _should_ be a default import. I left you a couple comments. Once those are address we will definitely merge! 🎉 

Thank you.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## 0.8.1 -2018-08-01
```
Comment (iansolano) [inline CHANGELOG.md]: can we bump this patch please

File: src/currency/index.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 // @flow
-import { Big } from 'big.js';
+//now use default import here (no brances) to support webpack
```
Comment (iansolano) [inline src/currency/index.js]: please remove this comment

File: CHANGELOG.md (hunk 1 above)
Comment (chrisgarber) [reply CHANGELOG.md]: Sorry, what changes did you want here? Did you want me to take these lines out?

**Comment by iansolano** [comment]
LGTM. Thank you for contributing!

**Pull request merged**